src/fragments/lib/graphqlapi/js/mutate-data.mdx

```diff
@@ -62,6 +62,10 @@ const deletedTodo = await API.graphql({ query: mutations.deleteTodo, variables:
 
 Only an `id` is needed.
 
+<Callout>
+In a many-to-many relationship, the children are not removed and you must explicitly delete them.
```

Comment:
Is this the exact wording we use in the datastore docs?

```suggestion
In a many-to-many relationship, the related items, and join records are not removed and you must explicitly delete them.
```

Reply:
Yes, it is from datastore docs. Anyways I've updated the wordings to match CPK docs PR.